### ISSUE TITLE
Network issues handling

### ISSUE THREAD

**Issue by LyashenkoGS** [LyashenkoGS/analytics4github#1]
Currently, the project relies on 
* [GitHub REST API ](https://developer.github.com/v3/)
* [GitHub trending page](https://github.com/trending) (parsing via JSoup)
Need to implement appropriate exceptions and their handling (show a message on UI) for cases of network connectivity issues.

Acception criteria
1. In case of absence of access to Github display the according error message
![image](https://cloud.githubusercontent.com/assets/12611051/21362728/4c353c5a-c6f2-11e6-88b6-b67808367900.png)
2.  In case of absence of access to Github Trending page,  display the according error message
![image](https://cloud.githubusercontent.com/assets/16464652/21363772/a12f93f0-c6f6-11e6-8d41-c11124df2bd9.png)

**Comment by LyashenkoGS** [comment]
Well done so far Nazar ! I finished your's  backend implementation and merged to the master. #9 
Left only displaying errors on UI.

**Comment by nazar-khimin** [comment]
After I analysed the accomplished work, I decided to remove  @ControllerAdvice and made exception handling locally in GithubTrendingController because the exception is thrown only once in the GithubTrendingController.

**Comment by nazar-khimin** [comment]
Handling Random Trending made.
![image](https://cloud.githubusercontent.com/assets/16464652/21930896/200207c8-d9a1-11e6-85ec-76c30832b8bd.png)

**Comment by azozello** [comment]
I`m going to change massages according to machups

**Comment by nazar-khimin** [comment]
The issue has been completed, http://134.249.116.246:8081/analytics-github/review/AG-CR-2

**Comment by nazar-khimin** [comment]
The origin/master merged into exceptionhandling.

**Comment by LyashenkoGS** [comment]
Merged the pull request #21 to the master branch. Good job @naz1719, my congratulations with the first completed issue.
